src/table/table.scss

```diff
@@ -122,265 +251,316 @@
-    height: 100%;
-    padding: $iui-xl;
+    align-items: center;
+    flex-basis: $iui-l * 2;
+
+    > .iui-icon {
+      @include themed {
+        fill: t(iui-icons-color-actionable);
+        color: t(iui-icons-color-actionable);
+      }
+      width: $iui-icons-default;
+      height: $iui-icons-default;
+      margin: 0;
+    }
+  }
+    
+  &.iui-right-align {
+    justify-content: flex-end;
   }
 
-  .iui-tables-paginator {
+  &.iui-positive {
     @include themed {
-      background-color: t(iui-color-background-3);
+      background-color: rgba(
+        t(iui-color-foreground-positive-rgb),
+        t(iui-opacity-6)
+      );
     }
-    user-select: none;
 
-    &:hover {
+    &::selection {
       @include themed {
-        background-color: t(iui-color-background-3) !important;
+        background-color: rgba(
+          t(iui-color-foreground-positive-rgb),
+          t(iui-opacity-4)
+        );
       }
     }
+  }
+    
+  &.iui-warning {
+    @include themed {
+      background-color: rgba(
+        t(iui-color-foreground-warning-rgb),
+        t(iui-opacity-6)
+      );
 
-    > .iui-content {
-      @include themed {
-        border-bottom: solid 1px t(iui-color-background-4);
-      }
-      width: 100%;
-      display: flex;
-      justify-content: space-around;
-      align-items: center;
-      padding: $iui-table-density;
-      flex: 1;
-
-      > .iui-full-line {
-        flex: 1;
-      }
-
-      > .iui-controls {
+      &::selection {
         @include themed {
-          color: t(iui-text-color);
-        }
-        width: 300px;
-        display: flex;
-        justify-content: center;
-        align-items: center;
-        flex: 1;
-
-        > a {
-          @include iui-anchor;
-          margin: 0 $iui-s;
-        }
-
-        > .iui-ellipsis {
-          margin: 0 $iui-s;
-        }
-
-        > .iui-current,
-        > .iui-ellipsis {
-          @include themed {
-            color: t(iui-text-color);
-          }
-          cursor: default;
+          background-color: rgba(
+            t(iui-color-foreground-warning-rgb),
+            t(iui-opacity-4)
+          );
         }
+      }
+    }
+  }
 
-        > .iui-current {
-          text-decoration: underline;
-          font-weight: $iui-font-weight-semibold;
-        }
+  &.iui-negative {
+    @include themed {
+      background-color: rgba(
+        t(iui-color-foreground-negative-rgb),
+        t(iui-opacity-6)
+      );
+    }
 
-        > a > .iui-icon {
-          @include themed {
-            color: t(iui-icons-color-actionable);
-            fill: t(iui-icons-color-actionable);
-          }
-          display: block;
-          width: $iui-icons-default;
-          height: $iui-icons-default;
+    &::selection {
+      @include themed {
+        background-color: rgba(
+          t(iui-color-foreground-negative-rgb),
+          t(iui-opacity-4)
+        );
+      }
+    }
+  }
 
-          cursor: pointer;
+  &.iui-editable {
```

Comment:
So this class adds a hover effect which is cool, but the focus effect is already broken.

What is the expected flow here? User clicks on this cell and it turns into a text input? Then user clicks outside this cell and it goes back to normal?

Reply:
Here's a GIF of how it works currently in Synchro's table.  If it wasn't a date picker I would imagine the cell could be manually typed into & clicking out of it would save the change.
![tablecell](https://user-images.githubusercontent.com/849817/119551305-7d83c180-bd67-11eb-950c-063c4fb6cf97.gif)

Reply:
If you click on a cell, the hover border disappears (thought that might be expected if we're gonna make it interactive).

Re: focus effect,

First cell does not have top border:
![image](https://user-images.githubusercontent.com/9084735/119552487-9b055b00-bd68-11eb-9ff8-5afac8cb8873.png)

Middle cells have uneven widths for top/bottom and right/left
![image](https://user-images.githubusercontent.com/9084735/119552809-f20b3000-bd68-11eb-888d-40d0ebd34260.png)

Last cell has thicker bottom than top:
![image](https://user-images.githubusercontent.com/9084735/119552871-00f1e280-bd69-11eb-90bd-56fc75428b79.png)

Reply:
It is broken currently..
Can I suggest we DO NOT include such things??
Can we somehow try to achieve now simple html structure with those basic styles (basically what we had) and then add up new things on it?
Editable is really specific again and lets work on it separately to make it look properly...